### DISCUSSION TITLE
JSON Modeling Techniques with dbt

### DISCUSSION THREAD

**Discussion by christineberger** [dbt-labs/devhub-writing#75] [Analytics craft]
How to model JSON?

This article aims to deep dive into JSON modeling and how & what you should be thinking about as you're considering the architecture needed to get to your end results.

[WIP Notion Page](https://www.notion.so/dbtlabs/JSON-Modeling-Techniques-dbt-acba1abb777d41938fd4c0317f8b1b6c)

**Comment by krevitt** [comment]
@christineberger that notion page looks pretty built out - think it's ready for an edit / to make its way towards publication?

**Reply by christineberger** [reply]
Thanks for checking on this! @sanjanasen96 had some ideas for splitting this up into different articles. I think we need to get together to figure out how we should roll that out and what additional content we need for each section!

**Reply by sanjanasen96** [reply]
@krevitt @jasnonaz 

@christineberger  and I jammed to split this into a three part blog series here: https://www.notion.so/dbtlabs/JSON-Article-Planning-ea488f97efff4474b95bd4a704809786

Could y'all take a look and let us know what you think?

**Reply by krevitt** [reply]
Huge fan of that DAG-like breakdown - do you think the 1st article would make sense as a glossary post (what is JSON + how is it relevant for data warehouses)?  Could build off of the [start here](https://www.notion.so/dbtlabs/656e06796224487fba289b718042f368?v=2bc132d3b4a94ab6b0d20b499a30d3cf&p=63d2c5ad79984eb6a1f523e2c5e30821) from office hours.

**Reply by johnblust** [reply]
@christineberger @sanjanasen96 I'm also a huge fan of the breakdown you all did. Agreed that the first part makes a lot of sense as a glossary post ([here's a WIP example](https://docs.google.com/document/d/1yUgZoJod8QB0D5Yp6hxf1QjFBYSadj2scyB75T-e2OE/edit#heading=h.2c91b2bpr8fp) of what that could look like). Whereas, pt 2 & 3 sound like amazing DevBlog posts.

Let me know if that sounds good & I can help you move on this any way I can :)!
